### DISCUSSION TITLE
waterfox g4 and ubuntu 21.10

### DISCUSSION THREAD

**Discussion by denikris** [hawkeye116477/waterfox-deb-rpm-arch-AppImage#41] [General]
hi,

there is new version of waterfox available and i need to install and update from apt for ubuntu 21.10
hope it'll be there soon, thanks

:-)

**Comment by hawkeye116477** [comment]
It will be soon, but be aware that in my opinion G4 looks like alpha/beta, not every option works same/proper like in G3.
That looks like someone hurried too much.

**Comment by hawkeye116477** [comment]
@denikris I've good news for you. G4 is now available on apt.

**Comment by mj162** [comment]
@hawkeye116477 Thank you so much for continuously building Waterfox as an AppImage.  I hope you're involved in knowledge transfer as (and you don't need me to tell you this) are so many projects that have not succeeded, or see it as too difficult.

I am using, and have been using for around a year, your Waterfox AppImage builds on HP ThinPro 7.1 (oem'd Ubuntu 16.04 LTS).  I check back every so often for G4 but appreciate that you're gifting your time to the community and I for one am grateful you do.
